docs/sync/azure_pipelines.md

```diff
@@ -0,0 +1,316 @@
+
+      # Ensure the latest Phylum CLI is installed.
+      - script: phylum-ci --force-install
+
+      # Install a specific version of the Phylum CLI.
+      - script: phylum-ci --phylum-release 3.10.0 --force-install
+
+      # Mix and match for your specific use case.
+      - script: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80 -al requirements-prod.txt
+```
+
+### Script Variables
+
+The script step environment variables are used to ensure the `phylum-ci` tool is able to perform it's job.
+
+A [Phylum token][phylum_tokens] with API access is required to perform analysis on project dependencies.
+[Contact Phylum][phylum_contact] or create an account and register to gain access.
+See also [`phylum auth register`][phylum_register] command documentation and consider using a bot or group account
+for this token.
+
+An Azure DevOps token with API access is required to use the API (e.g., to post notes/comments).
+This can be the default `System.AccessToken` provided automatically at the start of each pipeline build for the
+[scoped build identity][build_scope] or a personal access token (PAT).
+
+If using a PAT, it will need at least the `Pull Request Threads` scope (read & write).
+The account used to create the PAT will be the one that appears to post the comments on the pull request.
+Therefore, it might be worth using a bot or service account.
+See the Azure DevOps [documentation for using PATs to authenticate][azure_pat] for more info.
+
+If using the `System.AccessToken`, the [scoped build identity][build_scope] it attaches to needs at least the
+`Contribute to pull requests` permission. For example, to use the `System.AccessToken` on a project-scoped
+identity, follow these steps:
+
+* Go to project settings
+* Select the `Repos --> Repositories` menu
+* Select the `Security` tab
+* Select the user `{Project Name} Build Service ({Org Name})`
+  * NOTE: This user will only exist after the first time the pipeline has run
+* Ensure the `Contribute to pull requests` permission is set to `Allow`
+
+See the Azure DevOps documentation for [using the `System.AccessToken`][access_token] and setting it's
+[job authorization scope][auth_scope].
+
+Values for the `PHYLUM_API_KEY` and `AZURE_TOKEN` environment variable (e.g., `PHYLUM_TOKEN` and `AZURE_PAT`
+in the example here) can come from the pipeline UI, a variable group, or an Azure Key Vault. View the full
+[documentation for how to set secret variables][secret_variables] for more information.
+Since these tokens are sensitive, **care should be taken to protect them appropriately**.
+
+[azure_pat]: https://learn.microsoft.com/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+[secret_variables]: https://learn.microsoft.com/azure/devops/pipelines/process/set-secret-variables
+
+```yaml
+        env:
+          # Contact Phylum (https://phylum.io/contact-us/) or create an
+          # account and register to gain access. See also `phylum auth register`
```

Comment:
There is not a difference. This language is a holdover from the original GitHub Action IIRC. It will be changed to simply `register` since that is the link name on the main page for the "Sign Up" button - `https://app.phylum.io/register`